ui/src/geoviite-design-lib/split-button/split-button.scss

```diff
@@ -0,0 +1,18 @@
+.split-button {
+    display: inline-flex;
+
+    > :not(:last-child) {
+        margin-right: 0px;
+    }
+
+    > :first-child {
+        border-bottom-right-radius: 0px;
+        border-top-right-radius: 0px;
+    }
+
+    > :last-child {
+        border-bottom-left-radius: 0px;
+        border-top-left-radius: 0px;
+        border-left-width: 0px;
```

Comment:
Näissä ei sinänsä tarttis tuota px-suffiksia, en muista lähteekö nämä meidän buildin aikana jossain optimointistepissä veks (koska 0px == 0). Muistan että joskus on ollut toki näitä tapana laittaa/jättää, jotta dev-tooleilla saa kätevämmin kokeiltua muutoksia (kun unittia ei tarvii erikseen syöttää).

Reply:
Jep, nuo ei tosiaan mitenkään pakollisia ole. Olen itse tykännyt kirjoittaa ne, sillä silloin on selkeämpää missä yksikköavaruudessa ylipäätään tavallisesti näiden arvojen kanssa liikutaan

Reply:
Olen itse myös käyttänyt lintteriä joka nimenomaa nillitti jos noita ei merkannut. Varmaan siitä tottunut itsekin katsomaan tätä muotoa.